### ISSUE TITLE
Unable to swipe up view when in refresh state

### ISSUE THREAD

**Issue by Aaslte** [globulus/swiftui-pull-to-refresh#15]
Unable to swipe up view when in refresh state

**Comment by Aaslte** [comment]
```Swift
content
    .alignmentGuide(.top, computeValue: { _ in
        (state == .loading) ? -threshold + max(0, offset) : 0
     })
....
.offset(y: (state == .loading) ? -max(offset, 0) : -threshold)

```

**Comment by globulus** [comment]
Fixed the issue in v1.1.6. Thanks for reporting and for the fix!
